cmd/tacchaind/root.go

```diff
@@ -112,7 +114,16 @@ func NewRootCmd() *cobra.Command {
 			}
 
 			customAppTemplate, customAppConfig := initAppConfig()
-			customCMTConfig := initCometBFTConfig()
+
+			// Enforce faster block times
+			// 2 seconds + 1 second tendermint = 3 second blocks
+			timeoutCommit := 2 * time.Second
```

Comment:
can we add this as const in https://github.com/TacBuild/tacchain/blob/main/app/config.go please